src/dummy.erl

```diff
@@ -1 +1,3 @@
 -module(dummy).
+
+%% Long lineeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Line 3 is too long: %% Long lineeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%.